apps/vercel/frontend/src/clients/Vercel.ts

```diff
@@ -63,4 +108,20 @@ export default class VercelClient implements VercelAPIClient {
 
     return data;
   }
+
+  private filterServerlessFunctions(data: ServerlessFunction[]) {
+    return data.filter(
+      (file: ServerlessFunction) => file.type === 'Page' && file.path.includes('api')
+    );
+  }
+
+  private formatApiPaths(data: ServerlessFunction[]): ApiPath[] {
+    return data.map((file: ServerlessFunction) => {
+      const filePath = file.path;
+      return {
+        name: filePath,
+        id: filePath,
```

Comment:
there isn't an id associated with the `serverlessFunction` object returned, so these are the same thing for now. Thoughts?

Reply:
I think this is okay for now, maybe we can create a compound key for `id` with

```typescript
id: `${filePath}-${uuid()}`
``` 

Or something similar, we can brainstorm for sure.

Reply:
Ah yes, good idea. Will be sure to circle back to this 👍 I have added a note within the code here